### ISSUE TITLE
some problem with private repository,goimports: 10:2: invalid import path: "`gitlab.sogame.co/go/errors`" 

### ISSUE THREAD

**Issue by fighterlyt** [golangci/golangci-lint#1488]
Thank you for creating the issue!

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

Please include the following information:

<details><summary>Version of golangci-lint</summary>

```console
$ golangci-lint --version
# golangci-lint has version 1.32.1 built from dc2d6b5 on 2020-11-01T07:21:21Z
```

</details>

<details><summary>Config file</summary>

```console
$ cat .golangci.yml
# linters-settings:
  depguard:
    list-type: blacklist
    packages:
      # logging is allowed only by logutils.Log, logrus
      # is allowed to use only in logutils package
      - github.com/sirupsen/logrus
    packages-with-error-message:
      - github.com/sirupsen/logrus: "logging is allowed only by logutils.Log"
  dupl:
    threshold: 100
  funlen:
    lines: 100
    statements: 50
  goconst:
    min-len: 2
    min-occurrences: 2
  gocritic:
    enabled-tags:
      - diagnostic
      - experimental
      - opinionated
      - performance
      - style
    disabled-checks:
      - dupImport # https://github.com/go-critic/go-critic/issues/845
      - octalLiteral
      - wrapperFunc
  gocyclo:
    min-complexity: 15
  goimports:
    local-prefixes: github.com/golangci/golangci-lint
  golint:
    min-confidence: 0
  gomnd:
    settings:
      mnd:
        # don't include the "operation" and "assign"
        checks: argument,case,condition,return
  govet:
    check-shadowing: true
    settings:
      printf:
        funcs:
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Infof
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
  lll:
    line-length: 140
  maligned:
    suggest-new: true
  misspell:
    locale: US

linters:
  # please, do not use `enable-all`: it's deprecated and will be removed soon.
  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
  disable-all: true
  enable:
    - bodyclose
    - deadcode
    - depguard
    - dogsled
    - dupl
    - errcheck
    - funlen
    - gochecknoinits
    - goconst
    - gocyclo
    - gofmt
    - goimports
    - golint
    - gomnd
    - goprintffuncname
    - gosec
    - gosimple
    - govet
    - ineffassign
    - interfacer
    - lll
    - misspell
    - nakedret
    - rowserrcheck
    - scopelint
    - staticcheck
    - structcheck
    - stylecheck
    - typecheck
    - unconvert
    - unparam
    - unused
    - varcheck
    - whitespace
    - wsl

  # don't enable:
  # - gochecknoglobals
  # - gocognit
  # - godox
  # - maligned
  # - prealloc

issues:
  # Excluding configuration per-path, per-linter, per-text and per-source
  exclude-rules:
    - path: _test\.go
      linters:
        - gomnd

run:
  skip-dirs:
    - test/testdata_etc
    - internal/cache
    - internal/renameio
    - internal/robustio

# golangci.com configuration
# https://github.com/golangci/golangci/wiki/Configuration
service:
  golangci-lint-version: 1.23.x # use the fixed version to not introduce new linters unexpectedly
  prepare:
    - echo "here I can run custom commands, but no preparation needed for this repo"

```

</details>

<details><summary>Go environment</summary>

```console
$ go version && go env
# go version go1.15.3 darwin/amd64
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/liuyuntang/Library/Caches/go-build"
GOENV="/Users/liuyuntang/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOINSECURE=""
GOMODCACHE="/Users/liuyuntang/go/pkg/mod"
GONOPROXY="gitlab.sogame.co"
GONOSUMDB="gitlab.sogame.co"
GOOS="darwin"
GOPATH="/Users/liuyuntang/go"
GOPRIVATE="gitlab.sogame.co"
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/Cellar/go/1.15.3/libexec"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/Cellar/go/1.15.3/libexec/pkg/tool/darwin_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/Users/liuyuntang/projects/bank/abc/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/m0/c6dgtmld64j08nqt8hk15czc0000gn/T/go-build188307296=/tmp/go-build -gno-record-gcc-switches -fno-common"

```

</details>

<details><summary>Verbose output of running</summary>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
# INFO [config_reader] Config search paths: [./ /Users/liuyuntang/projects/bank/abc/protocol /Users/liuyuntang/projects/bank/abc /Users/liuyuntang/projects/bank /Users/liuyuntang/projects /Users/liuyuntang /Users /] 
INFO [config_reader] Used config file ../.golangci.yml 
INFO [lintersdb] Active 35 linters: [bodyclose deadcode depguard dogsled dupl errcheck funlen gochecknoinits goconst gocyclo gofmt goimports golint gomnd goprintffuncname gosec gosimple govet ineffassign interfacer lll misspell nakedret rowserrcheck scopelint staticcheck structcheck stylecheck typecheck unconvert unparam unused varcheck whitespace wsl] 
INFO [loader] Go packages loading at mode 575 (imports|types_sizes|deps|exports_file|files|name|compiled_files) took 243.717371ms 
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 889.187µs 
INFO [linters context/goanalysis] analyzers took 10.955184632s with top 10 stages: buildir: 7.467245164s, fact_deprecated: 624.883304ms, inspect: 580.878696ms, ctrlflow: 468.54795ms, printf: 327.532668ms, fact_purity: 277.496207ms, buildssa: 219.610817ms, gofmt: 195.385154ms, goimports: 178.440903ms, dupl: 129.915799ms 
WARN [runner] Can't run linter goanalysis_metalinter: goimports: 10:2: invalid import path: "`gitlab.sogame.co/go/errors`" 
INFO [linters context/goanalysis] analyzers took 26.550842ms with top 10 stages: buildir: 23.660625ms, U1000: 2.890217ms 
INFO [runner] Processors filtering stat (out/in): exclude: 7/7, nolint: 7/7, diff: 7/7, max_per_file_from_linter: 7/7, autogenerated_exclude: 7/7, skip_files: 7/7, skip_dirs: 7/7, exclude-rules: 7/7, uniq_by_line: 7/7, max_same_issues: 7/7, max_from_linter: 7/7, source_code: 7/7, path_prettifier: 7/7, path_shortener: 7/7, path_prefixer: 7/7, sort_results: 7/7, cgo: 7/7, identifier_marker: 7/7, severity-rules: 7/7, filename_unadjuster: 7/7 
INFO [runner] processing took 740.777µs with stages: nolint: 425.732µs, exclude: 122.336µs, path_prettifier: 65.589µs, autogenerated_exclude: 49.761µs, identifier_marker: 48.152µs, skip_dirs: 9.848µs, cgo: 3.158µs, max_per_file_from_linter: 2.947µs, max_same_issues: 2.813µs, filename_unadjuster: 2.331µs, uniq_by_line: 2.182µs, exclude-rules: 1.686µs, path_shortener: 1.23µs, max_from_linter: 915ns, source_code: 855ns, severity-rules: 461ns, diff: 269ns, skip_files: 214ns, sort_results: 183ns, path_prefixer: 115ns 
INFO [runner] linters took 3.354638877s with stages: goanalysis_metalinter: 3.22301181s, unused: 130.802074ms 
ERRO Running error: goimports: 10:2: invalid import path: "`gitlab.sogame.co/go/errors`" 
INFO Memory: 38 samples, avg is 416.3MB, max is 610.7MB 
INFO Execution took 3.609722174s          
```

</details>

**Comment by ernado** [comment]
Hey, can you please provide the way to reproduce this error? E.g. some minimal source file that triggers this issue, but is not breaking your NDA.

Looks like there is 
```go
import "`gitlab.sogame.co/go/errors`"
```
somewhere in you code :thinking:

**Comment by fighterlyt** [comment]
this is my private repository . the code can be compiled and worked

--

**Comment by bombsimon** [comment]
I can't reproduce this either, would you mind adding a minimal reproducible example?
